test/unit/rules/no-autofocus-attribute-test.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-autofocus-attribute',
+
+  config: true,
+
+  good: [
+    '<input />',
+    '<input type="text" disabled="true" />',
+    '<input type="password" disabled={{false}} />',
+    '{{input type="text" disabled=true}}',
+  ],
+
+  bad: [
```

Comment:
Would you mind adding some `good` and `bad` examples with elements _other_ than `<input />`? This should also include some examples using component invocation (e.g. `<AwesomeButton autofocus />`

Reply:
Additionally, since DOM attributes are case insensitive it would be good to have a `bad` example for `<input autoFocus>`